packages/devtools_app/lib/src/screens/logging/logging_screen_v2/logging_controller_v2.dart

```diff
@@ -634,15 +627,21 @@ class LogDataV2 with SearchableDataMixin {
 
   String? get details => _details;
 
-  bool get needsComputing => detailsComputer != null;
+  ValueListenable<bool> get needsComputing => _needsComputing;
+  final ValueNotifier<bool> _needsComputing = ValueNotifier<bool>(true);
```

Comment:
can we change this to be `computed` or `detailsComputed` with a default value of false? This would make the logic a bit easier to follow.